api/src/modules/sourcing-locations/materials-from-sourcing-locations.service.ts

```diff
@@ -0,0 +1,94 @@
+import { Injectable, NotFoundException } from '@nestjs/common';
```

Comment:
not sure we need another service for this. what's the added value vs using the existing one?